EasyCommands/Commands/Commands.cs

```diff
@@ -412,7 +410,7 @@ public class MultiActionCommand : Command {
                     }
                 }
 
-                if (currentCommands != null && currentCommands.Count > 0) return false;
+                if (currentCommands?.Count > 0) return false;
```

Comment:
is the ``?? 0``` needed here as well? Otherwise, is it not needed above?

Reply:
`null > 0` is `false` so giving a sensible default is not necessary here.